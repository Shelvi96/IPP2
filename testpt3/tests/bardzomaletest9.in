NEW ID

8:>740
956699>;18571
79>:;7
4;00180605>6;
38:7::>9;5
096>9;883
9;834>3:485570
9949>114711;11
::6:6>4108951
:>67571
8592:>3;312
40381382>2;2251703
@872;362
@;8;78
@71144667800:0419814
@4097:072880392;589
@4:46:0;791409494614890159
@87;6;74466
@555:95816488889896:43
@7987
@41;1:4:96897217228429:02
@19
@:8;292387291:60
@;0758
@8510;72110916;3:;2487668
