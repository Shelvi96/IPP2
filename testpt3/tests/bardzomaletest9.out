0
0
4096
3010
130111010000
0
0
0
12000001000
0
11
0
77507459712
